conformance/conformance_test.cc

```diff
@@ -675,6 +715,86 @@ bool ConformanceTestSuite::RunSuite(ConformanceTestRunner* runner,
     TestPrematureEOFForType(static_cast<FieldDescriptor::Type>(i));
   }
 
+  int64 kInt64Min = -9223372036854775808ULL;
```

Comment:
You would think so. However the `ULL` is applied before the `-` (the `-` is not actually part of the constant). And `ABS(kInt64Min)` is too large to fit in an int64, so this throws compiler warnings. Some authors get around this by writing it as: `-9223372036854775807LL - 1`.